docker/webui/run_openqa.sh

```diff
@@ -30,13 +30,14 @@ function webui() {
 }
 
 function all_together_apache() {
+  mode="prefork -m production"
   # run services
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler &
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets &
-  start_daemon -u geekotest /usr/share/openqa/script/openqa-livehandler &
+  start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler $mode &
+  start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets $mode &
+  start_daemon -u geekotest /usr/share/openqa/script/openqa-livehandler $mode &
```

Comment:
None of these daemons are allowed to run concurrently and therefore none of these daemons must run in prefork mode. Please just stick to what the scripts do and also what they don't do to avoid problems.